docs/DEPLOYMENT.md

```diff
@@ -1,32 +1,116 @@
+   7. For DB instance, we can go with Burstable classes to get started with. Select the one having vCPUs & RAM as per your needs.
+   8. Set the alloted storage as per your needs.
+   9. For Availability and Durability, use multi-AZ deployment if you need. Take a note that this may almost double your monthly expense.
+   10. Under VPC settings, select the VPC created in previous step.
+   11. In case you prefer to connect to your database directly from your machine, set the public access to "Yes".
+   12. Under additional configuration, set initial database name as `plio_staging`. This will create an empty database.
+   13. Finally, create the RDS instance by clicking on "Create Database" button.
+   14. You will see the new RDS instance in the list of RDS instances. Click on your DB instance to see the connectivity settings.
+   15. To connect to RDS from command line, run the following command (this will work if you've set public access to yes):
+   ```sh
+   psql -h your-db-instance-endpoint.aws-region.rds.amazonaws.com -p 5432 -U master_username
+   ```
+   16. Once you are logged in into the PSQL CLI, you can run all the SQL commands to create a new database, user, grant privileges etc.
+4. Create a new Elastic IP by going to EC2 dashboard and navigating to Elastic IP section.
+   1. Click on Allocate Elastic IP address and click on Allocate button.
+   2. You will see a new IP address in the IPs list. Name it `plio-backend-staging`.
+   3. This will be used in later steps to give the load balancer a permanent IP address.
+5. Go to Target groups.
+   1. Create a new target group.
+   2. Choose target type to be `IP addresses`.
+   3. Name the target group as `plio-backend-staging`.
+   4. Select the `plio-staging-vpc` for the target group VPC.
+   5. Proceed to create target group. You will see target group in the list.
+6. Go to Load Balancers.
+   1. Create a new load balancer.
+   2. Select Network Load Balancer option. We use NLB for easy support of web socket connections.
+   3. Name the LB as `plio-backend-staging`.
+   4. Select the `plio-staging-vpc` for the load balancer.
+   5. In the subnet mappings, check the first desired zone and use Elastic IP under IPv4 settings for that subnet.
+   6. Under listeners and routing, select the target group `plio-backend-staging` for TCP port 80.
+   7. Proceed to create load balancer. You will see load balancer in the list.
+7. Go to ECR and create a new repository named `plio-backend-staging`.
+8. Now go to ECS and create a new task definition with name `plio-backend-staging`.
+   1. Set the task role as `ecsTaskExecutionRole`.
+   2. Set the task memory and task CPU based on your needs.
+   3. Create a new container with name `plio-backend-staging`.
+   4. Enter port `80` in the port mapping field.
+   5. Use `.env.example` file to set all the required environment variables for your container.
+   6. Save the container definition and the task definition.
```

Comment:
The Google Client ID and Secret is generated from your personal account or from the Plio Google dev account?

Reply:
Hope you have updated it on staging.